docs/install/installing-istio.md

```diff
@@ -149,54 +166,20 @@ custom: custom-local-gateway
 If there is a change in service ports (compared to that of
 `knative-local-gateway`), update the port info in the gateway accordingly.
 
-### Verifying your Istio install
 
-View the status of your Istio installation to make sure the install was
+## Verifying your Istio installation
+
+View the status of your Istio installation to make sure the installation was
```

Comment:
The recent `istioctl` command has `verify-install` option so shall we change the docs to use it?
(I'm also fine for `kubectl get pods --namespace`, though.)